presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergNativeMetadata.java

```diff
@@ -323,12 +325,16 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
 
         PartitionSpec partitionSpec = parsePartitionFields(schema, getPartitioning(tableMetadata.getProperties()));
 
-        ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.builderWithExpectedSize(2);
+        ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.builderWithExpectedSize(3);
```

Comment:
nit: The `3` here looks like a magic number. Maybe we can refactor the number here?

Reply:
good call, I don't think we really need to maintain an "expected size", so I just removed it. Thanks!